diff --git a/node_modules/@mui/base/ClickAwayListener/ClickAwayListener.js b/node_modules/@mui/base/ClickAwayListener/ClickAwayListener.js
index ee3026b..7c3ad60 100644
--- a/node_modules/@mui/base/ClickAwayListener/ClickAwayListener.js
+++ b/node_modules/@mui/base/ClickAwayListener/ClickAwayListener.js
@@ -141,9 +141,9 @@ function ClickAwayListener(props) {
 
     return undefined;
   }, [handleClickAway, mouseEvent]);
-  return /*#__PURE__*/_jsx(React.Fragment, {
+  return React.createElement(React.Fragment, { children: _jsx(React.Fragment, {
     children: /*#__PURE__*/React.cloneElement(children, childrenProps)
-  });
+  }) });
 }
 
 process.env.NODE_ENV !== "production" ? ClickAwayListener.propTypes
