diff --git a/node_modules/@mui/material/CssBaseline/CssBaseline.js b/node_modules/@mui/material/CssBaseline/CssBaseline.js
index b660b79..281a9e6 100644
--- a/node_modules/@mui/material/CssBaseline/CssBaseline.js
+++ b/node_modules/@mui/material/CssBaseline/CssBaseline.js
@@ -69,11 +69,11 @@ function CssBaseline(inProps) {
     children,
     enableColorScheme = false
   } = props;
-  return /*#__PURE__*/_jsxs(React.Fragment, {
+  return React.createElement(React.Fragment, { children: _jsxs(React.Fragment, {
     children: [/*#__PURE__*/_jsx(GlobalStyles, {
       styles: theme => styles(theme, enableColorScheme)
     }), children]
-  });
+  }) });
 }
 
 process.env.NODE_ENV !== "production" ? CssBaseline.propTypes
